test/OpenTelemetry.Tests/Shared/DelegatingTestExporter.cs

```diff
@@ -15,32 +15,14 @@
 // </copyright>
 
 using System;
-using System.Collections.Generic;
 
 namespace OpenTelemetry.Tests
 {
     public class DelegatingTestExporter<T> : BaseExporter<T>
```

Comment:
Consider:
1. rename this to `DelegatingExporter` (it doesn't seem to have specific thing for test)
2. make it internal + sealed.